### DISCUSSION TITLE
How to move a branch to point to another commit?

### DISCUSSION THREAD

**Discussion by shauryagoel** [jesseduffield/lazygit#3516] [Q&A]
I want to move a branch to point to another commit. On the CLI, I use `git branch -f <branch> <commit_id>`.

**Comment by mark2185** [comment]
If it's to a specific branch, try pressing `g` or `u` in the `Local branches` panel, or if it's a specific commit try pressing `g` in the `Commits` panel.

**Reply by shauryagoel** [reply]
Are you suggesting to basically use `git reset` (as `g` opens up the reset panel)?

**Reply by mark2185** [reply]
I am, but I'm unsure what `git branch -f` does under the hood so YMMV.
